packages/text-editor/src/components/CollaboratorEditor.svelte

```diff
@@ -101,6 +101,8 @@
     provider?.on('synced', () => {
       loading = false
     })
+  } else {
+    loading = false
```

Comment:
do you use context provider?

Reply:
Yes, it's used through `Collaboration.svelte`

Reply:
Ah, ok, I overlooked this somehow. We need to investigate how to wait  for loading in this case then.